helix-core/src/syntax.rs

```diff
@@ -856,6 +886,14 @@ impl Syntax {
                         }
                     }
                 }
+
+                let hash = self.layers_hasher.hash_one(layer);
+                // Safety: insert_no_grow is unsafe because it assumes that the table
+                // has enough capacity to hold additional elements.
+                // This is always the case as we reserved enough capacity above.
+                unsafe {
+                    self.layers_tabel.insert_no_grow(hash, layer_id);
+                }
```

Comment:
```suggestion
                unsafe { self.layers_tabel.insert_no_grow(hash, layer_id) };
```

I think the line is short enough that `rustfmt` will make it a one-liner if the `;` is after the `unsafe` block